packages/create-sitecore-jss/src/templates/angular/package.json

```
@@ -35,7 +35,7 @@
     "postbuild:server": "move-cli ./dist/server/main.js ./dist/server.bundle.js && del-cli ./dist/server",
     "bootstrap": "ts-node --project src/tsconfig.webpack-server.json scripts/bootstrap.ts",
     "graphql:update": "ts-node --project src/tsconfig.webpack-server.json ./scripts/update-graphql-fragment-data.ts",
-    "install-pre-push-hook": "node ./scripts/install-pre-push-hook.js"
+    "install-pre-push-hook": "ts-node --project tsconfig.scripts.json ./scripts/install-pre-push-hook.ts"
```

Looks like tsconfig setup is different in angular. Shouldn't it be:
```suggestion
    "install-pre-push-hook": "ts-node --project src/tsconfig.webpack-server.json ./scripts/install-pre-push-hook.ts"
```